go/vt/vtctl/workflow/utils.go

```diff
@@ -627,9 +626,7 @@ func ReverseWorkflowName(workflow string) string {
 // this public, but it doesn't belong in package workflow. Maybe package sqltypes,
 // or maybe package sqlescape?
 func encodeString(in string) string {
```

Comment:
This identical `func` is defined in several files, should we make a common helper or just use `sqltypesEncodeStringSQL(...)` directly? 🤔

Reply:
Yeah, we can clean up these as well now.

Reply:
@timvaillancourt (and also @mattlord since you asked me about this too) Given that I aim to back port this, I think this is better / safer in a separate change. Making this change is a lot of churn / changes across many files which makes the back port harder.

So currently thinking of doing this cleanup then in a follow up PR that we don't need to back port.

Reply:
Sounds good!